README.md

```diff
@@ -1,171 +1,577 @@
-This set of libraries was built to help developers generate organic seed data for performance and unit testing. The following are descriptions of each library and what their purpose is.
+# Audacia.Seed
 
-## Audacia.Seed
+## Overview
+`Audacia.Seed` is an open-source library for generating test data. It can be used to build objects in memory, as well as seeding test data into a repository. It has support for the following ORMs:
```

Comment:
Is it worth also saying that it has support for seeding without an ORM? i.e. it can just generate objects for you without needing to put them in an EF `DbContext`?